PhotoApp/MainViewController/PopupViewController.swift

```diff
@@ -10,208 +10,222 @@ import UIKit
 import MapKit
 import Firebase
 
-
-
 extension String {
```

Comment:
@Kasiama important: **always move such generic extensions to separated files**

Reply:
fixed